pallene/coder.lua

```diff
@@ -1203,80 +1203,83 @@ gen_cmd["GetTable"] = function(self, cmd, _func)
     local key = self:c_value(cmd.src_k)
     local dst_typ = cmd.dst_typ
     local line = C.integer(cmd.loc.line)
```

Comment:
@hugomg Why do we also need to declare the `line` variable? It's used nowhere in the function.

Reply:
IIRC, GetTable needs the line info but SetTable doesn't. When we are reading a table field we also check if the type tag of the elemens is what we expected, and produce an error when it isn't.

Reply:
So you mean that the util.render() function always checks for line in its parameters in case of reading a table field?

Reply:
Oh, you are right. Now that the tag checking logic was moved to `get_luatable_slot`, the `util.render`  in `GetTable` is not using the `line` anymore. You can safely delete it.

Reply:
Got it. I Will remove it in a separate commit then start working on the other issue after this weekend.